lib/Onyx.js

```diff
@@ -48,12 +48,13 @@ const deferredInitTask = createDeferredTask();
  * Uses a selector string or function to return a simplified version of sourceData
  * @param {Mixed} sourceData
  * @param {String|Function} selector
+ * @param {Object} [withOnyxInstanceProps]
```

Comment:
Same as [this comment](https://github.com/Expensify/react-native-onyx/pull/247/files#r1186211659) should we update API.md to include that new param?